plugins/topology/README.md

```diff
@@ -1,44 +1,56 @@
 # Topology plugin for Backstage
 
-This plugin will help with visualizing the workloads like Deployment, Replicaset and Pods powering any service on the Kubernetes cluster.
+The Topology plugin helps with visualizing the workloads such as Deployment, Replicaset, and Pods powering any service on the Kubernetes cluster.
 
 ## Prerequisites
 
-1. Install and configure Kubernetes plugin by following these guides:
-   https://backstage.io/docs/features/kubernetes/installation
-   https://backstage.io/docs/features/kubernetes/configuration
-2. Kubernetes plugin is properly configured and able to connect to the cluster via a `ServiceAccount`
-3. Following [ClusterRole](https://backstage.io/docs/features/kubernetes/configuration#role-based-access-control) must be granted to ServiceAccount accessing the cluster (This role is already granted if you have the Backstage Kubernetes Plugin properly configured)
-4. To get the resources from a k8s cluster add the following annotations to the entity's catalog-info.yaml as K8s plugin identifies if the provided entity has k8s resources and if yes from which namespace it should get the resources based on the following annotations.
+1. Install and configure the Kubernetes plugin by following the [installation](https://backstage.io/docs/features/kubernetes/installation) and [configuration](https://backstage.io/docs/features/kubernetes/configuration) guides.
+2. The Kubernetes plugin is configured and able to connect to the cluster using a `ServiceAccount`.
```

Comment:
This is not a step. Maybe "Verify that the Kubernetes plugin..."?

Reply:
Also which particular Kubernetes plugin? Applies to the previous step and to this step.

Reply:
it's backstage k8s plugin both frontend and backend, details are mentioned in reference links in point 1. 

https://backstage.io/docs/features/kubernetes/installation
https://backstage.io/docs/features/kubernetes/configuration

Reply:
I suggest adding this information to this text. "Install and configure the Backstage Kubernetes plugin..."

Reply:
@mramendi the respective links talks about the steps for these in detail so can you elaborate on what shall we write more about here?

Reply:
@invincibleJai my suggestion is simply adding the name of the plugin to this phrase, so that people see this name without having to click the links.

Reply:
@mramendi you mean something like below

1. Install and configure the Kubernetes plugin (`@backstage/plugin-kubernetes` and `@backstage/plugin-kubernetes-backend`) by following the [installation](https://backstage.io/docs/features/kubernetes/installation) and [configuration](https://backstage.io/docs/features/kubernetes/configuration) guides.